notebooks/community/vision_ai/video_warehouse_sdk.ipynb

```diff
@@ -0,0 +1,820 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    PROJECT_NUMBER = int(PROJECT_NUMBER_STR[0])
</pre><p>use the template version</p><p> </p><p>change LOCATON_ID to REGION an use the template version</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2484/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>hi Andrew, I updated to REGION.</p><p> </p><p>However that PROJECT_NUMBER is different from PROJECT_ID which  I am not sure what you mean by "use the template version". Thanks!</p>

Reply:
As offline discussed, keep the project number here as it is, thanks!